manual/googlechrome-extensions/googlechrome-editthiscookie/googlechrome-editthiscookie.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>googlechrome-editthiscookie</id>
+    <version>1.0.0</version>
+    <title>EditThisCookie</title>
+    <authors>Francesco Capano</authors>
+    <owners>gep13 chocolatey</owners>
+    <projectUrl>http://www.editthiscookie.com/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/315c05f1387afb2acc1f028fd3752af86a9663f8/icons/googlechrome-editthiscookie.png</iconUrl>
+    <projectSourceUrl>https://github.com/fcapano/Edit-This-Cookie</projectSourceUrl>
+    <docsUrl>http://www.editthiscookie.com/start/</docsUrl>
+    <bugTrackerUrl>https://github.com/fcapano/Edit-This-Cookie/issues</bugTrackerUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/googlechrome-extensions/googlechrome-editthiscookie</packageSourceUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <licenseUrl>https://github.com/fcapano/Edit-This-Cookie/blob/master/License.txt</licenseUrl>
+    <description>
```

Comment:
Stick to [Informative description guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#123-informative-description) (Use `##Features` and `##Notes`)